### DISCUSSION TITLE
Wireguard on Raspberry 4 - Travel Access Point

### DISCUSSION THREAD

**Discussion by JamesWest1337** [RaspAP/raspap-webgui#1525] [Q&A]
Hello guys,
I would like to configure my PI4 as a mobile access point with a Wireguard connection to my home server.
All connections established via RaspAP's WiFi should be protected by a Wireguard tunnel. Is this possible?

Kind regards
James

**Comment by billz** [comment]
Yes, follow the detailed walkthrough (with video) https://docs.raspap.com/wireguard/#manual-configuration

**Comment by JamesWest1337** [comment]
Maybe I did not express myself correctly.
For example, I would like the PI4 to establish a Wireguard connection to my home server in a hotel (via eth0) and for the data traffic of all devices connected to the WiFi of the PI4 to be routed via the VPN tunnel.

**Reply by billz** [reply]
Create a peer configuration (same link, as above) specifying your home wg server's IP.

**Comment by CypressLo** [comment]
Hi @JamesWest1337 ,
I am trying to do the same thing with you. Did you work with creating a peer config in wireguard?
